### ISSUE TITLE
await fastify.view Cannot read property 'send' of undefined

### ISSUE THREAD

**Issue by momofan** [fastify/point-of-view#128]
Before you submit an issue we recommend you drop into the [Gitter community](https://gitter.im/fastify) or [Fastify Help](https://github.com/fastify/help) and ask any questions you have or mention any problems you've had getting started with Fastify.

**Please read this entire template before posting any issue. If you ignore these instructions
and post an issue here that does not follow the instructions, your issue might be closed,
locked, and assigned the `missing discussion` label.**

## 🐛 Bug Report

![image](https://user-images.githubusercontent.com/3773790/63086299-25a90b80-bf82-11e9-97c4-47ff269a0daa.png)
this.header return undefined
so "message": "Cannot read property 'send' of undefined"

change to:
this.header('Content-Type', 'text/html; charset=' + charset)
this.send(html)

## To Reproduce

Paste your code here:

```js
this.header('Content-Type', 'text/html; charset=' + charset)
this.send(html)
```

**Comment by mcollina** [comment]
Would you mind to send a Pull request?

**Comment by momofan** [comment]
> Would you mind to send a Pull request?

https://github.com/fastify/point-of-view/pull/129

**Comment by oguzhanoya** [comment]
I have the same problem.

**Comment by florianb** [comment]
:bell: I have the same issue using `ejs-mate` - is there anything i can do?

**Comment by Eomm** [comment]
Fixed in 3.5.1
